Fast and Flexible GPU Quantization for Transformers.md

```diff
@@ -301,6 +301,27 @@ Lastly we examine the effect of the aforementioned layout in memory on the matmu
 | INT8 COL32 (cuBLASLt) |    308    |   1.95x  |
 
 
+### The future of 8-bit quantization - FP8
+
+The arrival of Nvidia's Hopper/Lovelace architecture brought with it support for a new floating point datatype - FP8. This is available in two flavours:
+
+- **E5M2** - 5 exponent bits and 2 mantissa bits - larger dynamic range
+- **E4M3** - 4 exponent bits and 3 mantissa bits - higher precision 
+
+> Insert image here
+
+There are potential benefits to choosing FP8 as our quanitzation format from both an accuracy and performance perspective:
+
+##### Data distribution alignment
+When quantizing from FP16 to INT8 we not only reduce the range and number of values that can be represented, but also change the underlying distribution. Most of the tensors we want to quantize will be normally distributed, with more density around zero. This mirrors the representable floating point values - and is in contrast to the fixed point integers which provides a uniform distribution.  Research already suggests that we can remove/reduce the need for QAT (have we already defined this?) by using FP8 over INT8 (reference https://arxiv.org/abs/2208.09225 and https://arxiv.org/abs/2209.05433).
```

Comment:
Do you mind explaining that this is an example of non-uniform quantization? That will link nicely to the first section of the blog